app/scripts/router.js

```diff
@@ -104,6 +106,7 @@ function (
       'settings/avatar/gravatar(/)': showView(AvatarGravatarView),
       'settings/avatar/camera(/)': showView(AvatarCameraView),
       'settings/communication_preferences(/)': showView(CommunicationPreferencesView),
+      'settings/gravatar_permissions(/)': showView(GravatarPermissions),
```

Comment:
is it possible to have a better nested route?
something like `/settings/avatar/gravatar/permissions` or `/settings/avatar/gravatar_permissions`